src/binary/requests/state.ts

```diff
@@ -0,0 +1,14 @@
+import { StateType } from "../../consts";
+import { State } from "../state";
+import { getState } from "./requests";
+import { openConfigWithSource } from "../../commandsHandler";
+
+export default async function getStateA(): Promise<State | null | undefined> {
```

Comment:
change this to a meaningful name (its wip)

Reply:
you said so....